mesa/visualization/templates/js/PieChartModule.js

```diff
@@ -2,25 +2,25 @@
 //Note: This pie chart is based off the example found here:
 //https://bl.ocks.org/mbostock/3887235
 
-var PieChartModule = function(fields, canvas_width, canvas_height) {
+const PieChartModule = function(fields, canvas_width, canvas_height) {
     // Create the overall chart div
-    var chart_div_tag = "<div class='pie chart' width='" + canvas_width + "'></div>";
-    var chart_div = $(chart_div_tag)[0];
+    const chart_div_tag = "<div class='pie chart' width='" + canvas_width + "'></div>";
+    const chart_div = $(chart_div_tag)[0];
     $("#elements").append(chart_div);
 
     // Create the tag:
-    var svg_tag = "<svg width='" + canvas_width + "' height='" + canvas_height + "' ";
+    const svg_tag = "<svg width='" + canvas_width + "' height='" + canvas_height + "' ";
```

Comment:
This will cause error because of `svg_tag += ...` right after. The solution is to stick to `const`, and combine L12 and L13.